webapp/src/ts/modules/contacts/contacts-edit.component.ts

```diff
@@ -298,6 +302,9 @@ export class ContactsEditComponent implements OnInit, OnDestroy, AfterViewInit {
     if (this.enketoError) {
       this.globalActions.setEnketoError(null);
     }
+
+    this.duplicatesAcknowledged = false;
+    this.duplicates = [];
```

Comment:
This `resetFormError` will get called any time the user changes something in the form.  So, if they try to "Submit" and get a dupes error, then they go and change the `name` of the new contact they are creating, we want to clear out the duplicate state here so that we can come at it with a clean new check.